app/views/layouts/listnav/_network_service.html.haml

```diff
@@ -0,0 +1,16 @@
+- if @record.try(:name)
```

Comment:
This file should be obsolete now (as of https://github.com/ManageIQ/manageiq-ui-classic/pull/6997) - we've removed the listnav section from non-explorer views.